examples/messenger.js

```diff
@@ -102,15 +102,24 @@ const findOrCreateSession = (fbid) => {
 
 // Our bot actions
 const actions = {
-  send({sessionId}, {text}) {
+  send({sessionId}, response) {
     // Our bot has something to say!
     // Let's retrieve the Facebook user whose session belongs to
     const recipientId = sessions[sessionId].fbid;
     if (recipientId) {
       // Yay, we found our recipient!
+      
+      if (response.quickreplies) { // Wit.ai wants us to include quickreplies, alright!
+				response.quick_replies = []; // The quick reply object from Wit.ai needs to be renamed.
```

Comment:
Thank you for the requested changes, I really appreciate it. However, wouldn't creating a new variable (`data` in this case) maybe break in the future as you implement handling for CTAs etc? Or I might be confused and they wouldn't pass through the send action?